package.json

```diff
@@ -44,10 +44,13 @@
     "jest-chrome": "^0.7.2",
     "prettier": "^2.5.1",
     "ts-jest": "^27.1.3",
-    "tsup": "^5.12.8",
+    "tsup": "^6.2.2",
     "typescript": "^4.4.4"
   },
   "engines": {
     "node": ">= 12"
+  },
+  "dependencies": {
+    "regenerator-runtime": "^0.13.9"
```

Comment:
@victor-magalhaes-cb thoughts on making this a peer dependency? Maybe even optional with `peerDependenciesMeta ` if it's only required when using CJS?

Reply:
Seems good to me!